cou/utils/juju_utils.py

```diff
@@ -288,9 +303,16 @@ async def async_run_action(
     return action_obj
 
 
+# pylint: disable=too-many-arguments
 async def async_scp_from_unit(
-    unit_name, source, destination, model_name=None, user="ubuntu", proxy=False, scp_opts=""
-):
+    unit_name: str,
+    source: str,
+    destination: str,
+    model_name: Optional[str] = None,
+    user: str = "ubuntu",
+    proxy: bool = False,
+    scp_opts: str = "",
+) -> None:
     """Transfer files from unit_name in model_name.
 
     :param model_name: Name of model unit is in
```

Comment:
add Optional[str] to  model_name
add that user defaults to ubuntu
add that proxy defaults to False
add that scp_opts defaults to ""

Reply:
ok